src/app/application.cpp

```diff
@@ -139,11 +139,27 @@ void Application::torrentFinished(BitTorrent::TorrentHandle *const torrent)
 
     // AutoRun program
     if (pref->isAutoRunEnabled()) {
-        QString program = pref->getAutoRunProgram().trimmed();
-        // Replace %f by torrent path
-        program.replace("%f", torrent->savePathParsed());
-        // Replace %n by torrent name
+        QString program = pref->getAutoRunProgram();
+
+        int file_count = torrent->filesCount();
+
+        program.replace("%N", torrent->name());
         program.replace("%n", torrent->name());
```

Comment:
See the other comment. If we remove this in the future add a comment with the text //Deprecated or some explanation.